### ISSUE TITLE
Blank line rule on file beginning

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google/closure-linter#63]
```
> What steps will reproduce the problem?

1. Create a file that starts with:

/** ... */
function something() { ... }

2. Run gjslint on that file with --strict

> What is the expected output? What do you see instead?

I would expect no errors, but gjslint is complaining about:
Line 1, E:0007: Should have 2 blank lines between top-level blocks.

What version of the product are you using? On what operating system?

closure-linter=2.3.9 on Ubuntu 12.10


> Please provide any additional information below.

I think it would make sense to skip this error check "if line == 1"
```

Original issue reported on code.google.com by `Tobias.B...@googlemail.com` on 12 Mar 2013 at 8:33
